docs/faq.md

```diff
@@ -129,3 +129,22 @@ Run "docker system df" to see how much space can be reclaimed. If the space is l
 - (3) Check /tmp directory. 
 
 Run command "du -h / | awk '$1~/[0-9]*G/{print $0}'" to list all the directory which consumes more than 1G spaces. Clean the data not used any more. 
+
+### Q: How to change Docker install path for PAI?
+
+A: If we install Docker on a small disk, when under disk pressure, the cleaner may activate and kill jobs to prevent system crash. We'd better to move Docker to a larger disk. We can change Docker install path through following steps:
```

Comment:
avoid to use "we". It can be rephrased like,

If default Docker cache path is set to a disk with small disk space, the path can be changed with below steps.

Reply:
Done